### PR TITLE
Use native decorators + add some primitive tests for making sure the blueprints run

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: scripts/tests/test-addon-blueprint.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# -e - fails the script if any command errors
+#
+set -e
+
+npm install -g ember
```
Comment (rwjblue) [inline scripts/tests/test-addon-blueprint.sh]: This seems wrong (should be ember-cli), but you can avoid doing two things via:

    npx ember-cli addon my-addon -b $ADDON_PATH

File: scripts/tests/test-app-blueprint.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# -e - fails the script if any command errors
+#
+set -e
+
+npm install -g ember
```
Comment (rwjblue) [inline scripts/tests/test-app-blueprint.sh]: Same as above

File: scripts/tests/test-addon-blueprint.sh (hunk 1 above)
Comment (NullVoxPopuli) [reply scripts/tests/test-addon-blueprint.sh]: doh. of course it passes on my system, cause I already have e-c installed. thanks

**Comment by NullVoxPopuli** [comment]
@rwjblue  latest commit, should fix C.I. 

turns out, we can't install @glimmer/component via the addAddonsToProject hook, cause glimmer/component isn't an addon.

Good thing there are some tests now :)

**Comment by rwjblue** [comment]
it is definitely an addon, we need to install the most recent tag (an alpha I think) not the `latest` dist-tag. 

Specifically, we need @glimmer/component@^0.14.0-alpha.3.

**Comment by rwjblue** [comment]
We should fix it if it isn’t properly identifying as an addon though. Also, we should avoid using addAddonToProject from the blueprint: we are bringing our own package.json we should just have what we need listed there...

**Comment by NullVoxPopuli** [comment]
@rwjblue added the carets.

to keep the number of changes down in this PR, I'll defer removal of addAddonToProject for another time. :)

**Pull request merged**